drake/util/expmap2quat.h

```diff
@@ -1,8 +1,19 @@
 #include <Eigen/Dense>
 
-Eigen::Vector4d expmap2quatNonDegenerate(const Eigen::Ref<const Eigen::Vector3d>& v, double theta);
-Eigen::Vector4d expmap2quatDegenerate(const Eigen::Ref<const Eigen::Vector3d>& v, double theta);
-Eigen::Matrix<double, 4, 3> dexpmap2quatNonDegenerate(const Eigen::Ref<const Eigen::Vector3d>& v, double theta);
-Eigen::Matrix<double, 4, 3> dexpmap2quatDegenerate(const Eigen::Ref<const Eigen::Vector3d>& v, double theta);
-Eigen::Matrix<double, 12, 3> ddexpmap2quatNonDegenerate(const Eigen::Ref<const Eigen::Vector3d>& v, double theta);
-Eigen::Matrix<double, 12, 3> ddexpmap2quatDegenerate(const Eigen::Ref<const Eigen::Vector3d>& v, double theta);
+#undef DLLEXPORT
+#if defined(WIN32) || defined(WIN64)
+#if defined(drakeGeometryUtil_EXPORTS)
+#define DLLEXPORT __declspec( dllexport )
+#else
+#define DLLEXPORT __declspec( dllimport )
+#endif
+#else
+#define DLLEXPORT
+#endif
```

Comment:
have you thought about using a common header, like drakeGeometryUtilExports.h to store these definitions?